_config.yml

```diff
@@ -13,3 +13,29 @@ defaultid: 'mapbox.streets'
 future: true
 plugins:
 - jekyll-sitemap
+
+# This metadata is required on all pages
+defaults:
+  # Set products/services/platforms for every page
+  - scope:
+      path: ""
+    values:
+      products:
+        - Mapbox.js
+      services:
+        - maps
+      platforms:
+        - web
+  # Set contentType conditionally
+  - scope:
+      path: "_posts/examples"
+    values:
+      content_type: "example"
+  - scope:
+      path: "_posts/api"
+    values:
+      content_type: "API"
+  - scope:
+      path: "plugins"
+    values:
+      content_type: "example"
```

Comment:
Could consider tagging plugins as "reference" since they aren't the same kind of examples, but I don't feel strongly about that.

Reply:
Per voice, we are going to remove the content_type on the plugins page.